sjb/config/test_cases/test_branch_image_registry_extended.yml

```diff
@@ -1,5 +1,11 @@
 ---
 parent: 'common/test_cases/origin_built_installed_release.yml'
+overrides:
+  sync:
+    - "openshift,openshift-ansible=master"
```

Comment:
This is the correct approach, but we also want to list `openshift,origin` in here